lyte_helper.php_example

```diff
@@ -110,4 +110,11 @@ function fgo_lyte_change_css($lyte_css) {
 function no_lyte_on_mobile(){
 	return false;
 	}
+
+/** use internal lazy loader if you don't have another one configured on your site; the default for this filter is true */
+//add_filter('lyte_use_internal_lazyloader','__return_false');
+
+/** activate this filter if you have a lazy loader plugin on your site which can handle background images; the default for this filter is false */
```

Comment:
this isn't very useful any more is it? I mean if we have default LYTE behavior + your JS-based "true lazyload" approach, why bother with yet another one?

Reply:
True